provider/pkg/provider/provider.go

```diff
@@ -648,6 +656,31 @@ func (p *googleCloudProvider) getConfig(configName, envName string) string {
 	return os.Getenv(envName)
 }
 
+func (p *googleCloudProvider) getPartnerNumber() string {
+	customPartnerID := p.getConfig("partnerNumber", "GOOGLE_PARTNER_NUMBER")
+	if customPartnerID != "" {
+		return customPartnerID
+	} else {
+		disablePartnerID := p.getConfig("disablePartnerNumber", "GOOGLE_DISABLE_PARTNER_NUMBER")
+		if disablePartnerID == "true" {
+			return ""
+		}
+	}
+	return "Pulumi"
```

Comment:
It's a little odd to return a string for a "partner number". Is this correct?

Reply:
It is odd. They call it "google partner number" but just confirmed it should be "Pulumi" 🤷

Reply:
Okay, maybe it's actually "google partner name". I'll change the name in the PR.